frontend/packages/topology/src/components/quick-search/QuickSearchModalBody.tsx

```diff
@@ -139,7 +140,11 @@ const QuickSearchModalBody: React.FC<QuickSearchModalBodyProps> = ({
   }, [closeModal, onCancel, onEnter, selectNext, selectPrevious]);
 
   return (
-    <div ref={ref}>
+    <div
+      ref={ref}
+      className="odc-quick-search-modal-body"
+      style={{ height: catalogItems?.length > 0 ? '466px' : '60px' }}
```

Comment:
Something wrong with 466px as it always shows a scrollbar when there are 5 results.
On my browser the height is 467px.
Instead of 60px you can also use undefined.
Lastly, in react `'466px'` is equivalent to the number `466`. Prefer using numbers instead of strings.

Reply:
I kept the `60px` height because the height of the input field is `60px` and it is a child of modal body.